pkg/scheduler/framework/types.go

```diff
@@ -234,7 +234,11 @@ func (f *FitError) Error() string {
 	reasons := make(map[string]int)
 	for _, status := range f.Diagnosis.NodeToStatusMap {
 		for _, reason := range status.Reasons() {
-			reasons[reason]++
+			if len(status.failedPlugin) > 0 && klog.V(8).Enabled() {
```

Comment:
Maybe V(6)?

Reply:
But this not just affects the log, it also affects the Event message. I don't think we should change that.

Reply:
Got you, will just log the failed plugin for each node instead.